packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { useBreadcrumbsStyleProps } from './useBreadcrumbsStyleProps';
+import { SpiritBreadcrumbsProps } from '../../types';
+import { useStyleProps } from '../../hooks/styleProps';
+
+const defaultProps = {
+  goBackTitle: 'Back',
```

Comment:
I think that this default should also be empty. We do not want to handle any default strings or their translations. `goBackTitle` should be applied only when there is an`items` prop used. When using `children`, there should be no default or it should be empty.

Typescript cannot work simply with mutually exclusive types, which `children` and `items` in this case are. So we must let the developer decide which one he/she wants to use.

Please set the default to an empty string.

Reply:
Default value is empty string now.